### ISSUE TITLE
Beta8 NPE on start-up

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#2487]
### Description
NPE on start-up

### Steps to Reproduce
Update from 2.4.0-beta7 to 2.4.0-beta8
Start server

### Errors / Screenshots
```
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[18:38:29] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[18:38:29] [Server thread/ERROR]: #!#! You should report it at https://github.com/SkriptLang/Skript/issues. Please copy paste this report there (or use paste service).
[18:38:29] [Server thread/ERROR]: #!#! This ensures that your issue is noticed and will be fixed as soon as possible.
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Stack trace:
[18:38:29] [Server thread/ERROR]: #!#! java.lang.NullPointerException
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.entity.SheepData.init(SheepData.java:64)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.entity.SheepData.init(SheepData.java:1)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.entity.EntityData.getData(EntityData.java:507)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.entity.EntityData.fromClass(EntityData.java:523)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.expressions.ExprSpawnerType.<clinit>(ExprSpawnerType.java:63)
[18:38:29] [Server thread/ERROR]: #!#!     at java.base/java.lang.Class.forName0(Native Method)
[18:38:29] [Server thread/ERROR]: #!#!     at java.base/java.lang.Class.forName(Class.java:398)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptAddon.loadClasses(SkriptAddon.java:108)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript.onEnable(Skript.java:426)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:352)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:417)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:461)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:375)
[18:38:29] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:449)
[18:38:29] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:266)
[18:38:29] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:775)
[18:38:29] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:834)
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Version Information:
[18:38:29] [Server thread/ERROR]: #!#!   Skript: 2.4-beta8
[18:38:29] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
[18:38:29] [Server thread/ERROR]: #!#!     Date: 2019-09-29
[18:38:29] [Server thread/ERROR]: #!#!   Bukkit: 1.14.4-R0.1-SNAPSHOT
[18:38:29] [Server thread/ERROR]: #!#!   Minecraft: 1.14.4
[18:38:29] [Server thread/ERROR]: #!#!   Java: 11.0.4 (Java HotSpot(TM) 64-Bit Server VM 11.0.4+10-LTS)
[18:38:29] [Server thread/ERROR]: #!#!   OS: Linux amd64 3.10.0-1062.1.1.el7.x86_64
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Server platform: Spigot
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Current node: null
[18:38:29] [Server thread/ERROR]: #!#! Current item: null
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Thread: Server thread
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Language: english
[18:38:29] [Server thread/ERROR]: #!#! Link parse mode: DISABLED
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! End of Error.
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[18:38:29] [Server thread/ERROR]: #!#! Skript's class ch.njol.skript.expressions.ExprSpawnerType generated an exception while loading
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[18:38:29] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[18:38:29] [Server thread/ERROR]: #!#! You should report it at https://github.com/SkriptLang/Skript/issues. Please copy paste this report there (or use paste service).
[18:38:29] [Server thread/ERROR]: #!#! This ensures that your issue is noticed and will be fixed as soon as possible.
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Stack trace:
[18:38:29] [Server thread/ERROR]: #!#!   warning: no/empty exception given, dumping current stack trace instead
[18:38:29] [Server thread/ERROR]: #!#! java.lang.Exception: ch.njol.skript.util.EmptyStacktraceException
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript.exception(Skript.java:1546)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript.exception(Skript.java:1414)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptAddon.loadClasses(SkriptAddon.java:112)
[18:38:29] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript.onEnable(Skript.java:426)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:352)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:417)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:461)
[18:38:29] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:375)
[18:38:29] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:449)
[18:38:29] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:266)
[18:38:29] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:775)
[18:38:29] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:834)
[18:38:29] [Server thread/ERROR]: #!#! Caused by: ch.njol.skript.util.EmptyStacktraceException
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Version Information:
[18:38:29] [Server thread/ERROR]: #!#!   Skript: 2.4-beta8
[18:38:29] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
[18:38:29] [Server thread/ERROR]: #!#!     Date: 2019-09-29
[18:38:29] [Server thread/ERROR]: #!#!   Bukkit: 1.14.4-R0.1-SNAPSHOT
[18:38:29] [Server thread/ERROR]: #!#!   Minecraft: 1.14.4
[18:38:29] [Server thread/ERROR]: #!#!   Java: 11.0.4 (Java HotSpot(TM) 64-Bit Server VM 11.0.4+10-LTS)
[18:38:29] [Server thread/ERROR]: #!#!   OS: Linux amd64 3.10.0-1062.1.1.el7.x86_64
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Server platform: Spigot
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Current node: null
[18:38:29] [Server thread/ERROR]: #!#! Current item: null
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Thread: Server thread
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! Language: english
[18:38:29] [Server thread/ERROR]: #!#! Link parse mode: DISABLED
[18:38:29] [Server thread/ERROR]: #!#! 
[18:38:29] [Server thread/ERROR]: #!#! End of Error.
[18:38:29] [Server thread/ERROR]: #!#! 
```

**Comment by Wealthyturtle** [comment]
Oh no, my bad, sorry will fix in a PR for Sheep

**Comment by SlimeDog** [comment]
Confirmed fixed in 2.4-beta9. Thanks.
